### ISSUE TITLE
Link With Unclear Purpose fails even when aria-label is defined

### ISSUE THREAD

**Issue by gurmukhp** [GoogleChrome/accessibility-developer-tools#366]
Hi all,

I've noticed that the LinkWithUnclearPurpose test fails if there is no text inside the anchor element **and** if aria-label has been specified.

I believe that this test should pass since the label has been defined.

If you agree, I'd be more than happy to provide a PR?

Thanks!

**Comment by steventai22** [comment]
How can close my issues
